docs/reference/commandline/daemon.md

```diff
@@ -603,6 +603,10 @@ The currently supported cluster store options are:
     private key is used as the client key for communication with the
     Key/Value store.
 
+*  `kv.path`
+
+    Specifies the path you want to use in the Key/Value store. (default: docker/nodes)
```

Comment:
Specifies the path in the Key/Value store. If not configured, the default value is 'docker/nodes'.